config/locales/en.yml

```diff
@@ -0,0 +1,69 @@
+en:
+  menu:
+    start: 'start'
+    rules: 'rules'
+    statistics: 'statistics'
+    exit: 'exit'
+    agree: 'y'
+    disagree: 'n'
+    info: |
+      Enter one of next commands:
+          start -> to start game
+          rules -> to show rules for game
+          statistics -> to show statistics
+          exit  -> to exit from console
+  registration:
+    name: "Enter name:\n"
+  error:
+    command_error: 'Wrong command. Try again!'
+    guess_input_error: "The guess must be 4 digits long.\nEach digit is a number nust be in the range 1-6:\n"
+    name_length_error: "The name must be from 3 to 20 characters long.\n"
+    difficulty_input_error: "This difficulty is not available.\n"
+  game:
+    guess: "Enter guess: (write hint to receive hint, write exit to leave game)\n"
+    hint: 'hint'
+    matrix: "Matrix: %{matrix}"
+    no_hints: 'You have no hints'
+    loss: "Secret code: %{code}\nYou lost!\n"
+    win: "You won!\n"
+    save_result: "Save results? (y/n):\n"
+    new_game: "Start a new game? (y/n):\n"
+    stats: 'Players rating'
+    exit: 'Goodbye!'
+    difficulty: |
+      Enter difficulty:
+        easy -> 15 attempts and 2 hints
+        medium -> 10 attempts and 1 hint
+        hell -> 5 attempts and 1 hint
```

Comment:
I guess we have to get info about difficulty from codebraker gem. What if difficulty in codebraker will be changed?